gensim/models/doc2vec.py

```diff
@@ -21,17 +21,21 @@
 >>> model.save(fname)
 >>> model = Doc2Vec.load(fname)  # you can continue training with the loaded model!
 
-The word vectors in the model can also be instantiated from an existing file on disk in the word2vec C format:
-NOTE: document vectors are not loaded/saved with .load/save_word2vec_format(). Use .save()/.load() instead.
+If you're finished training a model (=no more updates, only querying), you can do
+
+  >>> model.delete_temporary_training_data(keep_doctags_vectors=True, keep_inference=True):
```

Comment:
Inference is preserved. It is tested in https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/test/test_doc2vec.py#L319